README.md

```diff
@@ -55,7 +55,7 @@ A web-based application to understand about the basics of React - Redux
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- See it live by clicking [Live Demo Link]()
+- See it live by clicking [Live Demo Link](https://metrics-webapp-4iuc.onrender.com/)
```

Comment:
- [ ] Well done styling your App however as requested before you still have 2 issues to work on:
-  Record a video presenting the features of the project. with a maximum  length of 5 minutes. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. The video contains a demo of the project, highlighting all the project features and a highlight about a piece of the code the student finds interesting or is proud of making.

- [ ] You project is still. not responsive (you still have the white space around for bigger screens):

<img width="571" alt="Screenshot 2023-08-05 at 9 22 50 PM" src="https://github.com/Kidd254/Metrics-WebApp-React-Redux/assets/53347966/7f65f28c-14fd-4b1c-8eb7-7fa060d59192">